wp-content/themes/pub/wporg-learn-2024/inc/block-config.php

```diff
@@ -557,3 +559,41 @@ function inject_other_filters( $key ) {
 		printf( '<input type="hidden" name="s" value="%s" />', esc_attr( $wp_query->query['s'] ) );
 	}
 }
+
+/**
+ * Modify the course query on the 'My Courses' page to display courses according to the filter status.
+ * Corresponds to https://github.com/Automattic/sensei/blob/trunk/includes/blocks/course-list/class-sensei-course-list-student-course-filter.php#L95
+ *
+ * @param array $query The course query.
+ * @return array The modified course query.
+ */
+function modify_course_query( $query ) {
```

Comment:
Now that this is just a query modification I think it would be better in `inc/query.php`

Reply:
Actually ignore that, I can see this is closely related to the other filters in here.